README.md

```diff
@@ -33,7 +33,7 @@ Optionally, you may hand over the `--repository` (or short `-r`) flag to specify
 $ get-next-version --repository <PATH>
 ```
 
-If you need to prefix the version, you can use the `--prefix` (or short `-p`) flag.
+If you need to prefix the version, you can use the `--prefix` (or short `-p`) flag. Note that the prefix must be a valid tag name on its own (see [implementation comments](git/tags.go#L65) for details).
```

Comment:
While the idea of having a reference to the file (and even the line) is basically great, it doesn't work out in the long run. The next commit (which maybe is completely unrelated to this one) may change the line number, or even the file's name or path – and I almost certainly can guarantee that whoever will do this change won't think of adjusting this comment in the `README.md` file.

So, while I like the sentence you added, I'd suggest to remove the part within the parentheses.

Reply:
I see the problem you are addressing. It kicks me out of my perfect world where everyone is using the refactor function of all those wonderful Czech IDEs.